config/repos.yaml

```diff
@@ -11,36 +11,45 @@ theforeman/chef-handler-foreman:
   redmine: chef
 theforeman/community-templates:
   redmine: foreman
+  prprocessor: true
```

Comment:
You added this because it was already there in practice but not configured?

Reply:
I started adding it with the idea to track this but I should remove it.